source/SIL.AppBuilder.Portal/src/routes/(authenticated)/projects/[filter=projectSelector]/[id]/+page.svelte

```diff
@@ -1,8 +1,96 @@
 <script lang="ts">
-  import ProjectSelector from '../../components/ProjectSelector.svelte';
   import type { PageData } from './$types';
+  import { goto } from '$app/navigation';
+  import { page } from '$app/stores';
+  import IconContainer from '$lib/components/IconContainer.svelte';
+  import * as m from '$lib/paraglide/messages';
+  import ProjectFilterSelector from '$lib/projects/components/ProjectFilterSelector.svelte';
+  import ProjectCard from '$lib/projects/components/ProjectCard.svelte';
 
   export let data: PageData;
+
+  let selectedProjects: number[] = [];
+  let selectedOrg = parseInt($page.params.id);
+  let searchTerm: string = '';
+
+  $: filteredProjects = data.projects.filter((project) => {
+    const searchTermLower = searchTerm.toLowerCase();
+    return (
+      project.Name?.toLowerCase().includes(searchTermLower.toLowerCase()) ||
+      project.Language?.toLowerCase().includes(searchTermLower) ||
+      project.OwnerName?.toLowerCase().includes(searchTermLower) ||
+      project.OrganizationName?.toLowerCase().includes(searchTermLower) ||
+      project.GroupName?.toLowerCase().includes(searchTermLower)
+    );
```

Comment:
```suggestion
    return [
      project.Name,
      project.Language,
      project.OwnerName,
      project.OrganizationName,
      project.GroupName
    ].some((field) => field?.toLowerCase().includes(searchTermLower));
```

Reply:
Didn't know you could do it like that. Added to both `/projects` and `/directory`